main.py

```
@@ -1,44 +1,437 @@
+        )
+        markup = telebot.types.InlineKeyboardMarkup()
+        markup.add(telebot.types.InlineKeyboardButton("💎 VIP Aktivasiya", callback_data="show_rules"))
+        bot.send_message(target_id, msg, reply_markup=markup, disable_web_page_preview=True)
+        bot.reply_to(message, "✅ Test prosesi tamamlandı")
+    except Exception as e:
+        bot.reply_to(message, f"❌ Xəta: {e}\nFormat: /test ID VAXT")
+
+@bot.message_handler(content_types=['photo'])
+def handle_proofs(message):
+    uid = str(message.chat.id)
+    if is_banned(uid): return
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)